requirements.txt

```diff
@@ -11,3 +11,4 @@ requests-oauthlib==0.7.0
 six==1.10.0
 tweepy==3.8.0
 wheel==0.24.0
+psycopg2==2.9.1
```

Comment:
Fact that we're switching to PG, aren't we also supposed to remove the MySQL related stuff? `mysqlclient`, `mysql_track`.py, etc.

Reply:
Yes. Although, I think the reason they wanted a `track-module` is to be able to pull data(accounts) from different databases, and a developer only needed to create a track module for that specific database.

Reply:
🤔 ... If they were going for multi-database architecture, wouldn't there be different politwoops-worker.py  as well? e.g. one for MySQL and developers can add another one for PG, etc.